app/core/firebaseData.service.js

```diff
@@ -0,0 +1,17 @@
+(function() {
+  'use strict';
+
+  angular
+    .module('megaNote.core')
+    .factory('firebaseDataService', firebaseDataService);
+
+  function firebaseDataService() {
+    var root = firebase.database().ref();
```

Comment:
My code complained that firebase is not defined... What do you think was the issue?